src/equisolve/utils/metrics.py

```diff
@@ -0,0 +1,14 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2023 Authors and contributors
+# (see the AUTHORS.rst file for the full list of names)
+#
+# Released under the BSD 3-Clause "New" or "Revised" License
+# SPDX-License-Identifier: BSD-3-Clause
+import numpy as np
+from equistore import TensorMap
+
+
+def rmse(y_true: TensorMap, y_pred: TensorMap) -> float:
+    """TODO: Needs to be a tensormap implementation."""
```

Comment:
@agoscinski ;-)

Reply:
For now cannot just do:
- check if the metadata are the same 
- ``` return np.sqrt(np.mean((y_true.values - y_pred.values) ** 2))```
- do the same for the grad
It works only if the metadata are in the same order but it is simple

Am I not getting something?

Reply:
Yes, more or less. I will write the function now...